src/js/lib/monster.ui.js

```diff
@@ -275,6 +275,15 @@ define(function(require) {
 				return monster.ui.getTemplatePhoneNumber(number.toString());
 			},
 
+			renderSvgIcon: function(id, cssClass) {
+				var template = monster.ui.getTemplateSvgIcon({
+					id: id,
+					cssClass: _.isString(cssClass) ? cssClass : undefined
+				});
+
+				return new Handlebars.SafeString(template);
```

Comment:
If this is just a wrapper of `getTemplateSvgIcon` I think this line should be moved to `monster.ui.getTemplateSvgIcon` and just return his result:
```
return monster.ui.getTemplateSvgIcon({
  id: id,
  cssClass: _.isString(cssClass) ? cssClass : undefined
});
```

let me know if i'm wrong :)

Reply:
I was considering it, but looking at the code in monster.ui.js, only the handlebars helpers create `Handlebars.SafeString` objects, so I think it can be kept this way.